android/src/main/java/bbct/android/common/activity/MainActivity.java

```diff
@@ -69,13 +69,16 @@ public void onCreate(Bundle savedInstanceState) {
                     BaseballCardContract.PROJECTION, null, null, null);
 
             FragmentTransaction ft = this.getSupportFragmentManager().beginTransaction();
+            ft.add(R.id.fragment_holder, new BaseballCardList(), FragmentTags.CARD_LIST)
+                    .addToBackStack(null)
+                    .commit();
+
             if (cursor == null || cursor.getCount() == 0) {
-                ft.add(R.id.fragment_holder, new BaseballCardDetails(), FragmentTags.EDIT_CARD);
-            } else {
-                ft.add(R.id.fragment_holder, new BaseballCardList(), FragmentTags.CARD_LIST);
-                cursor.close();
+                this.getSupportFragmentManager().beginTransaction()
+                        .replace(R.id.fragment_holder, new BaseballCardDetails(), FragmentTags.EDIT_CARD)
+                        .addToBackStack(FragmentTags.EDIT_CARD).commit();
             }
-            ft.commit();
+            cursor.close();
```

Comment:
The same goes for all the changes in this file.

Reply:
I assumed we would go back to the BaseballCardList when we pressback from BaseballCardDetails. Isn't that right?

Reply:
Yes, that is correct. But since this exact change already exists in PR #428, it should not be repeated here. Instead, it should be merged in from that branch. I will make time to finish reviewing the tests in #428 this week and hopefully get it merged into `android/devel`.

Reply:
Okay, then I can merge with that then update this PR right?

Reply:
Let's put this PR on hold for now until I merge #428. After that, the best solution will probably be a rebase.